src/Config.php

```diff
@@ -285,27 +285,23 @@ private static function init()
 
         switch (PHP_OS_FAMILY) {
             case "Windows":
-                $library_ext = ".dll";
+                // win needs the ABI number before the .dll
+                $library_ext = "42.dll";
```

Comment:
On Windows this is called `libvips-42.dll`.
```suggestion
                $library_ext = "-42.dll";
```
(untested)